packages/prisma/schema.prisma

```diff
@@ -281,6 +281,7 @@ model Team {
   slug                String?
   logo                String?
   logoUrl             String?
+  orgCalVideoLogo     String?
```

Comment:
I named it `orgCalVideoLogo` because only organization have custom cal video logo and not any Team. calVideoLogo sounds much better. I'll update this.